app/data/schedule.js

```diff
@@ -2,17 +2,27 @@ const moment = require('moment');
 
 const schedule = [
   {
-    start: moment('2017-08-24T13:00+01'),
-    end: moment('2017-08-25T12:59:59+01'),
-    resource: 'featured03',
+    start: moment('2017-09-01T13:00+01'),
+    end: moment('2017-09-04T12:59:59+01'),
+    resource: 'featured02',
   }, {
-    start: moment('2017-08-25T13:00+01'),
-    end: moment('2017-08-29T12:59:59+01'),
+    start: moment('2017-09-04T13:00+01'),
+    end: moment('2017-09-05T12:59:59+01'),
+    resource: 'featured03',
+  },
+  {
+    start: moment('2017-09-05T13:00+01'),
+    end: moment('2017-09-06T12:59:59+01'),
+    resource: 'featured04',
+  },
+  {
+    start: moment('2017-09-06T13:00+01'),
+    end: moment('2017-09-07T12:59:59+01'),
     resource: 'featured05',
```

Comment:
What do you want displayed between 07-08?

Reply:
Missed a date, good spot. It's fixed now.